examples/exp_configs/rl/multiagent/multiagent_i210.py

```diff
@@ -54,11 +56,24 @@
     num_vehicles=0,
     lane_change_params=SumoLaneChangeParams(lane_change_mode="strategic"),
     acceleration_controller=(IDMController, {"a": .3, "b": 2.0, "noise": 0.5}),
+    car_following_params=SumoCarFollowingParams(speed_mode="no_collide"),
 )
 vehicles.add(
     "av",
     acceleration_controller=(RLController, {}),
     num_vehicles=0,
+    color='red'
+)
+
+vehicles.add(
+    "aggressive",
+    acceleration_controller=(IDMController, {"a": 3.0, "b": 8.0, "noise": 0.5, "T": 0.0, "s0": 0.0}),
+    car_following_params=SumoCarFollowingParams(speed_mode='no_collide'),
+    lane_change_params=SumoLaneChangeParams(lane_change_mode="no_lat_collide"),
+    lane_change_controller=(SafeAggressiveLaneChanger, {
+                            "target_velocity": 100.0, "threshold": 1.0, "desired_lc_time_headway": 0.1}),
+    num_vehicles=0,
+    color='green',
```

Comment:
Err, how can we confirm that these are good or correct settings? This seems something specific to your project.

Reply:
Hmm let me see if theres a way to move these into a different file / area. I can also explicitly not include the aggressive driver stuff since it's not directly related to the i210 work